arm-sql/2015-05-01/swagger/sql.json

```diff
@@ -0,0 +1,227 @@
+            },
+            "get": {
+                "tags": [
+                    "SecurityAlertPolicy"
+                ],
+                "operationId": "SecurityAlertPolicy_GetServerSecurityAlertPolicy",
+                "description": "Returns an Azure SQL Server security alert policy.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ServerNameParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The Server Threat Detection Policy",
+                        "schema": {
+                            "$ref": "#/definitions/ServerSecurityAlertPolicy"
+                        }
+                    },
+                    "400": {
+                        "description": "Bad Request"
+                    },
+                    "404": {
+                        "description": "Resource was not found"
+                    }
+                }
+            }
+        }
+    },
+  "definitions": {
+    "ServerSecurityAlertPolicyProperties": {
+      "properties": {
+        "state": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The state of the Azure SQL Security alert policy (New, Enabled or Disabled)."
+        },
+        "disabledAlerts": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The disable alerts of the Azure SQL Security alert policy."
+        },
+        "emailAddresses": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The list of email addresses to send the alert."
+        },
+        "emailAccountAdmins": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Value that indicated whether to email the azure account admins."
+        }
+      },
+      "description": "Represents the properties of an Azure SQL Security alert policy."
+    },
+    "ServerSecurityAlertPolicyCreateOrUpdateParameters": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ServerSecurityAlertPolicyProperties",
+          "description": "Properties of the request."
+        }
+      },
+      "required": [
+        "properties"
+      ],
+      "description": "Create or update server security alert policy parameters."
+    },
+    "ServerSecurityAlertPolicy": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ServerSecurityAlertPolicyProperties",
+          "description": "Represents the properties of the resource."
+        }
```

Comment:
Please look at the concept of Resource Flattening over here: https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/creating-swagger.md#ResourceFlattening